docs/go/editors/vim.md

```diff
@@ -40,6 +40,11 @@ autocompletion for Bazel-generated Go files, among other things.
 
       " See https://github.com/golang/tools/blob/master/gopls/doc/settings.md
       let g:go_gopls_settings = {
+        \ 'build.workspaceFiles': [
+          \ '**/BUILD',
+          \ '**/WORKSPACE',
+          \ '**/*.{bzl,bazel}',
```

Comment:
Why does this include arbitrary `.bzl` files? It would seem more correct to me to instead list `MODULE.bazel` and `REPO.bazel` explicitly.

Reply:
I see, I think I misunderstood the feature at first. Looks good!